### PR TITLE
8.0 base partner sequence new api

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: base_partner_sequence/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================
+Add a sequence on customers' code
```
Comment (elicoidal) [inline base_partner_sequence/README.rst]: s/customer/partner

File: base_partner_sequence/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+=================================
+
+This module adds the possibility to define a sequence for
+the partner code. This code is then set as default when you
+create a new commercial partner, using the defined sequence.
```
Comment (elicoidal) [inline base_partner_sequence/README.rst]: remove commercial (or you mean company?).

File: base_partner_sequence/tests/test_base_partner_sequence.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     abi_connector_akeneo is free software:
```
Comment (sbidoul) [inline base_partner_sequence/tests/test_base_partner_sequence.py]: wrong copy/paste of header :wink:

File: base_partner_sequence/tests/test_base_partner_sequence.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline base_partner_sequence/tests/test_base_partner_sequence.py:1]: simplify header following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

File: base_partner_sequence/tests/test_base_partner_sequence.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class TestBasePartnerSequence(common.TransactionCase):
+
+    def test_ref_on_partner_supplier(self):
+        res_partner = self.env['res.partner']
+        partner = res_partner.create({
+            'name': 'test1',
+            'email': "test@test.com"})
+        self.assertTrue(partner.ref, 'A simple partner has always a ref')
```
Comment (elicoidal) [inline base_partner_sequence/tests/test_base_partner_sequence.py]: add period

File: base_partner_sequence/tests/test_base_partner_sequence.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+        copy = partner.copy()
+        self.assertTrue(copy.ref, 'A partner with ref created by copy '
+                        'has a ref by default')
```
Comment (elicoidal) [inline base_partner_sequence/tests/test_base_partner_sequence.py]: add period

File: base_partner_sequence/tests/test_base_partner_sequence.py (hunk 4 above)
Comment (lmignon) [inline base_partner_sequence/tests/test_base_partner_sequence.py:1]: @elicoidal no guidelines for headers in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

**Comment by lmignon** [comment]
@elicoida, @sbidoul: done

File: base_partner_sequence/tests/test_base_partner_sequence.py (hunk 4 above)
Comment (elicoidal) [inline base_partner_sequence/tests/test_base_partner_sequence.py:1]: @lmignon oups. I was sure about that!
Sorry: I am quite sure I have seen some conversation about the topic and took it for granted.

**Comment by Cedric-Pigeon** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**